build-wallet-electron.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+# downloads the bitcoin-s-server.zip file corresponding to your operating system 
+# then builds the Suredbits Wallet app so it can be run in dev mode 
+
+OS="`uname`"
+case $OS in
+  'Linux')
+   OS='Linux'
+   alias ls='ls --color=auto'
+    ;;
+  'FreeBSD')
+    OS='FreeBSD'
+    alias ls='ls -G'
+    ;;
+  'WindowsNT')
+    OS='Windows'
+    ;;
+  'Darwin') 
+    OS='Mac'
+    ;;
+  'arm64')
+    OS='Mac'
+    ;;
+  'SunOS')
+    OS='Solaris'
+    ;;
+  'AIX') ;;
+  *) ;;
+esac
+
+cd wallet-electron-ts
+
+if [[ $OS == 'Mac' ]]; then 
+	curl -O -L https://github.com/bitcoin-s/bitcoin-s/releases/download/1.9.2/bitcoin-s-server-mac.os.x-1.9.2.zip
+elif [[ $OS == 'Windows' ]]; then
+	curl -O -L https://github.com/bitcoin-s/bitcoin-s/releases/download/1.9.2/bitcoin-s-server-windows.server.2019-1.9.2.zip
+elif [[ $OS == 'Linux' ]]; then
+	curl -O -L https://github.com/bitcoin-s/bitcoin-s/releases/download/1.9.2/bitcoin-s-server-linux-1.9.2.zip
+else 
+	echo "not supported OS"
+fi
+
+for name in *bitcoin-s-server*; do
+	if [ -f "$name" ]; then 
+		cd ..
+		npm i && npm run build 
```

Comment:
Does this complain the second time it runs on a Mac? I think you probably need to do a `npm run clean` before doing a second `npm run build`